CS342MemberViews/Package.swift

```diff
@@ -77,6 +78,19 @@ let package = Package(
             resources: [
                 .process("Resources")
             ]
-            )
+        ),
+        .target(
+            name: "RajPabari",
+            dependencies: [
+                .target(name: "InstructorViews")
+            ],
+            exclude: [
+                "Resources/RajPabari.jpeg.license",
+                "Resources/RajPabariBio.md.license"
+            ],
+            resources: [
+                .process("Resources")
+            ]
+        )      
```

Comment:
Please remove the training whitespaces to fix your SwiftLint error:
```suggestion
        )
```